add_me_to_your_PATH/sysimage.jl

```diff
@@ -25,8 +28,8 @@ function main()
     packages = setdiff(packages, exclude)
     println("Creating sys image with deps being tracked from a registry:")
     println(packages)
-    create_sysimage(packages; sysimage_path="deps.so", cpu_target = "generic;sandybridge,-xsaveopt,clone_all;haswell,-rdrnd,base(1)")
-    Pkg.rm("PackageCompiler")
+    create_sysimage(packages; replace_default=true, cpu_target = "generic;sandybridge,-xsaveopt,clone_all;haswell,-rdrnd,base(1)")
```

Comment:
Why replace the default? It seems confusing to me if julia_pod changes stuff outside of the pod environment (and can cause issues with versioning, eg you can’t trust your manifest if you’re using a custom sysimage built in a different environment)

Reply:
`sysimage.jl` should only ever get run in the context of the docker build, which is private to each manifest/julia env; this won't affect the julia environment outside of that context. In fact, julia_pod doesn't assume you have julia installed locally at all.

The main thing this does I'm guessing (@omus might have other motivations?) is make the `CMD julia` that gets called at the end of the `Dockerfile.template` simpler.

I think with this, we no longer need the `[[ -n "${JULIA_COMMAND}" ]]` block in the `julia_pod` script, since the julia command is just `julia` in all cases.

Reply:
The motivation behind replacing the default system image was that it plays nicer when we fully skip making a system image (due to having no pinned packages). In that scenario the original `/deps.so` file would not exist causing any call of `julia --sysimage=/deps.so` to fail. I could copy the original system image to this location but replacement seems like a better option. Do note that PackageCompiler backs up the previous system image so you can always restore the original

Reply:
Ah ok, I didn’t realize that this is only inside the docker env. 👍